docs/help/policies/sla.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: "Cloud Standard, Professional, & Enterprise Edition: Hasura Service Level Agreement"
+description: Hasura Service Level Agreement for Hasura Cloud
+sidebar_label: Hasura SLA
+sidebar_position: 2
+keywords:
+  - hasura
+  - service level agreement
+  - SLA
+sidebar_class_name: cloud-and-enterprise-icon
+---
+
+# Hasura DDN Service Level Agreement (SLA)
+
+## Introduction
+
+This document describes the Service Level Agreement (SLA) calculation and crediting process for Hasura DDN. The SLA
+applies to paid tiers in Hasura DDN.
+
+## Service Availability
+
+We calculate SLA uptime monthly and measure uptime with an HTTP request to the `/v1/version` endpoint on a project at a
+regular interval.
+
+DDN Base users can find more detail regarding uptime in our
+[terms of service](https://hasura.io/legal/hasura-cloud-terms-of-service/). Enterprise users should reference their
+contracts for details.
+
+## Credits
+
+If the SLA for your tier is not met, you are eligible for credit. We calculate the credit amount as a percentage of the
+monthly subscription fee for affected Hasura DDN projects.
+
+To request credits, you must notify the [Hasura Support team](https://cloud.hasura.io/support/create-ticket) within
+thirty days from when you become eligible to receive credit. Failure to comply with this requirement will forfeit your
+right to receive credit.
```

Comment:
@shahidhk is issuing a credit for a SLA miss something we've done before and is the support team aware of it?